src/leappto/cli.py

```diff
@@ -384,7 +384,16 @@ def _virt_tar_out():
                 return _rsync()
             return _virt_tar_out()
 
-        def check_target(self):
+        def check_service_status(self, name, check_cmd):
+            rc = self._ssh_sudo(check_cmd)
+            print("! Check {} availability at {}: {}".format(
```

Comment:
This looks a bit confusing, the parameters to format are vertically close to `print`, but still not quite there with the first argument, I'd suggest changing the code so that the argument to `format` are aligned with `format` in pep8 manner:

```python
msg = "! Check {} availability at {}: {}"
print(msg.format(name, self.target.hostname,
                 "success" if not rc else "error"))
```